packages/demo/examples/08-network/renderer/AttributeNode.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const proptypes = {
+  nodeStyles: PropTypes.object,
+  node: PropTypes.object.isRequired,
```

Comment:
should almost never use object because it's not specific, can you update to shape? (could define a shape in `@data-ui/network` and import it)